basics/signals/README.md

```diff
@@ -51,32 +55,32 @@ Let's see how we can implement this ...
 ## A simple React button
 
 We start with a file called `src/widget.tsx`. The `tsx` extension allows to use
-XML-like syntax with the tag notation `<>`to represent some visual elements
-(note that you might have to add a line: `"jsx": "react",` to the
-`tsconfig.json` file).
+HTML-like syntax with the tag notation `<>`to represent some visual elements
+(note that you have to add a line: `"jsx": "react",` to the
+`tsconfig.json` file). This is a special syntax used by [React](https://reactjs.org/tutorial/tutorial.html).
 
 `widget.tsx` contains one major class `TutorialView` that extends the
-`VDomRendered` class that is provided by Jupyterlab. `VDomRenderer` defines a
-`render()` method that defines some html elements (react) such as a button.
+`ReactWidget` class provided by Jupyterlab. `ReactWidget` defines a
+`render()` method that defines some React elements such as a button. In
+this the recommanded way to include React component inside the Jupyterlab widget based
```

Comment:
Minor typo spotted:

```suggestion
this the recommended way to include React component inside the JupyterLab widget based
```

Also should the beginning of the sentence be: "This is the recommended"?

Reply:
Thanks!